src/Components/Browser.JS/src/components.server.css

```diff
@@ -0,0 +1,14 @@
+.modal {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 1000;
+  display: none;
+  overflow: hidden;
+  background-color: #fff;
+  opacity: 0.8;
+  text-align: center;
+  font-weight: bold;
+}
```

Comment:
My preference would be to avoid adding an extra CSS file because

1. It's another concept for users to understand in the index page (or to start debates about how exactly it should be included, bundled, versioned, etc.)
2. It's another HTTP request that shows up in the network tab

Instead I'd prefer to inline these styles directly into the code in `AutoReconnectCircuitHandler.ts`, e.g.:

```ts
this.modal = document.createElement('div');
this.modal.setAttribute('style', 'position: fixed; top: 0; ... etc ...');
```

I know that makes it harder to override, but people still *can* override them either by:

 * Replacing `AutoReconnectCircuitHandler` entirely with their own thing
 * Or, just using `!important` on rules for `.modal` in their own `.css` files

Reply:
Seeing this comment, I think I like the proposal I made on AutoReconnectCircuitHandler even more. 
We don't get this file it's obvious for the consumer what's going on in their markup and makes it trivial to integrate with most frameworks. We can provide the default styles in the Index.cshtml if we are using bootstrap